roles/satellite-clone/tasks/backup_check.yml

```diff
@@ -97,4 +91,4 @@
       Standard backup scenario - config_files.tar.gz, mongo_data.tar.gz, pgsql_data.tar.gz, (optional) pulp_data.tar
       Online backup scenario - config_files.tar.gz, mongo_dump folder, foreman.dump, candlepin.dump, (optional) pulp_data.tar
```

Comment:
this row and the next needs to be merged now as online backup and rhel6 to 7 migration backup are using similar backup files.